src/N3Quoter.php

```diff
@@ -40,7 +40,49 @@ public function escapeIRI( $iri ) {
 	}
 
 	public function escapeLiteral( $s ) {
-		$escaped = addcslashes( $s, "\x0..\x1F\"\\" );
+		// String escapes. Note that the N3 spec is more restrictive than the Turtle and TR
+		// specifications, see <https://www.w3.org/TeamSubmission/n3/#escaping>
+		// and <https://www.w3.org/TR/turtle/#string>
+		// and <https://www.w3.org/TR/n-triples/#grammar-production-literal>.
```

Comment:
This says: ECHAR	::=	'\' [tbnrf"'\]

So there are more acceptable escapes than r. n and t - at least b and f also supported.